Shared/FeatureSwitch.swift

```diff
@@ -36,6 +36,12 @@ open class FeatureSwitch {
             return nonChannelValue
         }
 
+        // Check if this feature has been enabled by the user
+        let key = "feature_switches/\(self.featureID)_enabled"
```

Comment:
Nit: put this in a `fileprivate` calculated property.

Also; could you change `feature_switches/` to `feature_switches.` (replace `/` with `.`)? I made a mistake initially – but this should be isomorphic with `prefs.branch("feature_switches")`.